AsyncSwift/Observed/AppData.swift

```diff
@@ -6,16 +6,37 @@
 //
 
 import UIKit
+import SwiftUI
```

Comment:
```suggestion
import SwiftUI
import UIKit
```

So ,,, 사소하지만 ,,,,,, 알파벳순서,,,중요해요 ,,,

Reply:
수정하겠습니다!